cluster/src/main/java/io/scalecube/cluster/ClusterImpl.java

```diff
@@ -67,7 +67,8 @@
 
   // Disposables
   private final Disposable.Composite actionsDisposables = Disposables.composite();
-
```

Comment:
Why you rearrange fields? Can switch off fields rearrangement if it's setup in your IDEA; or if u do this manually, then just dont do that w/o some good reason (for example `static final` field in the middle of the class or misplaced constants).